usaspending_api/search/delta_models/award_search.py

```diff
@@ -635,12 +639,15 @@
             THEN SORT_ARRAY(COLLECT_SET(faba.disaster_emergency_fund_code))
         ELSE NULL
     END AS disaster_emergency_fund_codes,
-    COLLECT_SET(taa.treasury_account_identifier) AS treasury_account_identifiers
+    COLLECT_SET(taa.treasury_account_identifier) AS treasury_account_identifiers,
+    SORT_ARRAY(COLLECT_SET(rpa.program_activity_name), TRUE) AS program_activity_names,
```

Comment:
The default for `asc` parameter in `SORT_ARRAY` is `True`, so it is not strictly necessary to include it here.

Reply:
Yep, these were leftover from some testing I was doing. I've removed them to match our other uses of the `SORT_ARRAY` function.